docker/flower.override.yml

```diff
@@ -0,0 +1,26 @@
+version: "3"
+
+services:
+  rabbitmq:
+    image: library/rabbitmq:management
+    container_name: intelowl_rabbitmq
+    logging:
+      driver: none
+
+  flower:
+    image: intelowlproject/intelowl:${INTELOWL_TAG_VERSION}
+    container_name: intelowl_flower
+    restart: unless-stopped
+    stop_grace_period: 3m
+    volumes:
+      - generic_logs:/var/log/intel_owl
+    expose:
+      - "5555"
+    ports:
```

Comment:
shouldn't the ports be set on the `nginx` service?

Reply:
You are absolutely correct